src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/search/PrisonerOffenderSearchClient.kt

```diff
@@ -24,15 +23,13 @@ class PrisonerOffenderSearchClient(
 ) {
```

Comment:
Remove ^^^ `FIXME: PUD-1577  ...` comment above - assuming it is fixed by this PR.